test/components/border_box_component_test.rb

```diff
@@ -95,7 +95,7 @@ def test_strips_denied_system_arguments
   def test_renders_row_with_schemes
     { neutral: "gray", info: "blue", warning: "yellow" }.each_pair do |scheme, color|
       render_inline(Primer::BorderBoxComponent.new) do |component|
-        component.row(scheme: scheme) { "Row row row your boat" }
+        component.row(scheme: scheme) { "Row, row, row your boat" }
```

Comment:
Happy to drop, but technically this is the standard form for the song.

A recent addition to my tooling objects to `row row` as it's often an error. Quite happy to drop this or any other change.

Reply:
🤣  this is pretty funny, but acceptable.